nbs_probabilistic/create_dropout_ensemble.py

```diff
@@ -0,0 +1,145 @@
+import fire
+from fire import Fire
+import xarray as xr
+import numpy as np
+from src.data_generator import *
+from src.train import *
+from src.networks import *
+from src.utils import *
+from tensorflow.keras import backend as K
+#ToDO: 
+#make it work for all networks. #(Differences: custom_objects, loss function,  
+#load full data instead of batches. output for full size of X. sometimes stop due to RAM limits.
+#should we pass optional arguments. like is_normalized, start_date, end_date
+#what to do if output vars are different?? numpy-->xarray wont work
+
+def get_input(args):
+    #Get Arguments
+    #essential
+    exp_id=args['exp_id']
+    var_dict=args['var_dict']
+    batch_size=args['batch_size']
+    output_vars=args['output_vars'] #ToDo: need to change numpy-->xarray function if this changes.
+    test_years=args['test_years']
+    #optional. 
+    #Question: how to optionally input data_subsample, norm_subsample, nt_in, dt_in,test_years?
+    data_subsample=args['data_subsample']
+    norm_subsample=args['norm_subsample']
+    nt_in=args['nt_in']
+    dt_in=args['dt_in']
+    #changing paths
+    #model_save_dir='/../../data/WeatherBench/predictions/saved_models'
+    #datadir='/../../data/WeatherBench/5.625deg'
+      
+    #For Model. Ques: Needed?? Since we are not building the model again, but just simply switching on dropout, i dont think we need these.
+    loss=args['loss'] #How to incorporate this in model?
+    filters=args['filters']
+    kernels=args['kernels']
+    lead_time=args['lead_time']
+    lr=args['lr']
+    early_stopping_patience=args['early_stopping_patience']
+    reduce_lr_patience=args['reduce_lr_patience']
+    data_subsample=args['data_subsample']
+    norm_subsample=args['norm_subsample']
+    bn_position=args['bn_position']
+    dropout=args['dropout']
+    l2=args['l2']
+    nt_in=args['nt_in'] #Ques: Is nt and nt_in the same thing? important since it casuses problem in input size for model layer 1 sometimes.
+    #nt_in=args['nt']
+    activation=args['activation']
+    min_es_delta=args['min_es_delta'] #not for all configs.
+    
+    
+    #run data generator
+    ds = xr.merge([xr.open_mfdataset(f'../../data/WeatherBench/5.625deg/{var}/*.nc',
+                                     combine='by_coords') for var in var_dict.keys()])
+    mean = xr.open_dataarray(f'../../data/WeatherBench/predictions/saved_models/{exp_id}_mean.nc') 
+    std = xr.open_dataarray(f'../../data/WeatherBench/predictions/saved_models/{exp_id}_std.nc')
```

Comment:
The paths should not be hard coded, i.e. "../../data/WeatherBench/5.625deg" should be a parameter you pass to the script.